integration-tests/suites/runtime_config_file.go

```diff
@@ -0,0 +1,189 @@
+package suites
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
+	"path/filepath"
+	"time"
+
+	"github.com/stackrox/collector/integration-tests/pkg/assert"
+	"github.com/stackrox/collector/integration-tests/pkg/collector"
+	"github.com/stackrox/collector/integration-tests/pkg/common"
+	"github.com/stackrox/collector/integration-tests/pkg/config"
+	"github.com/stackrox/collector/integration-tests/pkg/types"
+)
+
+var (
+	normalizedIp = "255.255.255.255"
+	externalIp   = "8.8.8.8"
+	serverPort   = 53
+	externalUrl  = fmt.Sprintf("http://%s:%d", externalIp, serverPort)
+
+	activeNormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", normalizedIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: types.NilTimestamp,
+	}
+
+	activeUnnormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", externalIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: types.NilTimestamp,
+	}
+
+	inactiveNormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", normalizedIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: "Not nill time",
+	}
+
+	inactiveUnnormalizedConnection = types.NetworkInfo{
+		LocalAddress:   "",
+		RemoteAddress:  fmt.Sprintf("%s:%d", externalIp, serverPort),
+		Role:           "ROLE_CLIENT",
+		SocketFamily:   "SOCKET_FAMILY_UNKNOWN",
+		CloseTimestamp: "Not nill time",
+	}
+
+	runtimeConfigDir  = "/tmp/collector-test"
+	runtimeConfigFile = filepath.Join(runtimeConfigDir, "/runtime_config.yaml")
+	collectorIP       = "localhost"
+)
+
+type RuntimeConfigFileTestSuite struct {
+	IntegrationTestSuiteBase
+	ClientContainer string
+}
+
+func (s *RuntimeConfigFileTestSuite) setRuntimeConfig(runtimeConfigFile string, configStr string) {
+	cmd := "echo '" + configStr + "' > " + runtimeConfigFile
+	s.execShellCommand(cmd)
+}
+
+func (s *RuntimeConfigFileTestSuite) setExternalIpsEnable(runtimeConfigFile string, enable bool) {
```

Comment:
This might sound like a nit, but from a separation of concerns perspective, this function shouldn't do the marshalling - it should be handled by `setRuntimeConfig` 

This function should just construct `types.RuntimeConfig` object and pass it to `setRuntimeConfig`. Otherwise, when we inevitably come to adding another field to these tests, we'll have to duplicate the marshaling logic.

Reply:
Done